infra/Pos/DHT/Constants.hs

```diff
@@ -18,6 +18,6 @@ neighborsSendThreshold =
 kademliaDumpInterval :: Integral a => a
 kademliaDumpInterval = fromIntegral . ccKademliaDumpInterval $ infraConstants
 
--- | Setting this to true enables enhanced message broadcast
-enhancedMessageBroadcast :: Integral a => a
-enhancedMessageBroadcast = fromIntegral . ccEnhancedMessageBroadcast $ infraConstants
+-- | Broadcast to nodes which timestamp at most @enhancedMessageTimeout@.
```

Comment:
The "which" makes it confusing. Perhaps you meant "whose", i.e. "nodes whose timestamp is at most @enhancedMessageTimeout@", but then you'd have to explain what a node's timestamp being at most `x` means, this because the value in `constants-wallet-prod.yaml` is simply `10` - it's not immediate to see what's going on here.

Can you make this part a bit clearer?